### DISCUSSION TITLE
How to add an icon in NavigationView of FluentAvaloniaUI

### DISCUSSION THREAD

**Discussion by Daniil-Kolbasov** [AvaloniaUI/Avalonia#12557] [Q&A] [answered]
In example [FluentAvalonia.FluentIcons](https://github.com/taj-ny/FluentAvalonia.FluentIcons), the following code is specified:

```xaml
<Window xmlns="https://github.com/avaloniaui"
        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
        xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
        xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
        xmlns:ic="using:FluentAvalonia.FluentIcons"
        xmlns:ui="using:FluentAvalonia.UI.Controls"
        mc:Ignorable="d" d:DesignWidth="800" d:DesignHeight="450"
        x:Class="Avalonia.NETCoreApp1.MainWindow"
        Title="Avalonia.NETCoreApp1">
        
    <ui:NavigationView>
        <ui:NavigationView.MenuItems>
            <ui:NavigationViewItem Content="Page 1">
                <ui:NavigationViewItem.Icon>
                    <ic:FluentIcon Icon="Settings16Filled"/>
                </ui:NavigationViewItem.Icon>
            </ui:NavigationViewItem>

            <!-- The library also provides markup extensions to reduce boilerplate code -->
            <ui:NavigationViewItem Content="Page 2"
                                   Icon="{ic:FluentIcon Settings16Regular}"/>
        </ui:NavigationView.MenuItems>
    </ui:NavigationView>
</Window>
```

This cat causes an error when copying and pasting: Unable to resolve type FluentIcon from namespace using:FluentAvalonia.FluentIcons

![image](https://github.com/AvaloniaUI/Avalonia/assets/34882813/003a7294-acb0-4a87-80b4-3c4cfe215990)

And if you look in IntelliSense, then NavigationView does not have Icon properties, only IconSource. I also tried replacing using:FluentAvalonia.FluentIcons with clr-namespace:FluentAvalonia.FluentIcons, didn't work

**Comment by timunie** [comment, marked as answer]
Already told you: https://github.com/amwx/FluentAvalonia/issues/462#issuecomment-1680030321

The lib is not up to date, so you cannot use it as you have a version conflict. Wait for `[FluentAvalonia.FluentIcons](https://github.com/taj-ny/FluentAvalonia.FluentIcons)` to update or build the project on your own using the target version you have.
